modules/tinymce/changelog.txt

```diff
@@ -1,4 +1,5 @@
 Version 5.6.0 (TBD)
+    Added support for `block` and `unblock` methods on inline dialogs
```

Comment:
JIRA number is missing from this changelog entry

Reply:
Fixed